frontend/package.json

```diff
@@ -37,8 +37,8 @@
     "storybook": "storybook dev -p 6006 -s ../config",
     "cypress:open": "cypress open --project src/__tests__/cypress",
     "cypress:run": "cypress run -b chrome --project src/__tests__/cypress",
-    "cypress:open:mock": "MOCK=1 WS_PORT=9002 npm run cypress:open",
-    "cypress:run:mock": "MOCK=1 WS_PORT=9002 npm run cypress:run",
+    "cypress:open:mock": "MOCK=1 WS_PORT=9002 npm run cypress:open --",
+    "cypress:run:mock": "MOCK=1 WS_PORT=9002 npm run cypress:run --",
```

Comment:
This change seems unrelated.

Reply:
Ahh yeah, I brought that in to make my life easier... I'll remove it.